themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -99,6 +99,21 @@
           VuFind.setCspNonce('{$cspNonce}');
           VuFind.setCurrentSearchId($searchId);
           JS;
+
+      if (!empty($lightboxParent)) {
+        $lightboxParent = json_encode((string)$lightboxParent);
+        $appendScripts[] = <<<JS
+                   VuFind.lightbox.parent = {$lightboxParent};
```

Comment:
Since these new scripts are just one-liners, maybe it would be cleaner to do it with a single quoted string instead of HEREDOC.

Reply:
done